test/responses_unit_test.go

```diff
@@ -109,18 +107,40 @@ func weMakeAnyCallTo(client /* algod/indexer */, endpoint string) (err error) {
 				GenesisID:        sParams.GenesisID,
 				Genesishash:      sParams.GenesisHash,
 				LastRound:        uint64(sParams.FirstRoundValid),
-				MinFee:           81560,
+				MinFee:           sParams.MinFee,
```

Comment:
Apparently the old test input only had a single value... that is no longer true so I needed to add MinFee to SuggestedParams